### PR TITLE
Fix misusing the days since last inactivity

### LINKED ISSUE #146: The bot is mixing misusing the days since last inactivity
**Describe the issue**

When the bot creates an inactivity issue its using days of inactivity for the repo where it means to use the number of days to trigger an issue. For example, in this line of text it should be using `INACTIVE_DAYS` (180) rather than `459.13`. Also, for bonus points, it might be worth mentioning they can add a `dormant` or `retired` life cycle badge:

https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md

```
To help differentiate products that are unmaintained from products that do not require frequent maintenance,
repomountie will open an issue whenever a repository has not been updated in 459.13 days.
```

**Additional context**

Check this example out:
https://github.com/bcgov/cap-eagle-mobile-inspections/issues/6

**Definition of done**

- [ ] In the second sentence it uses the `INACTIVE_DAYS` in the template rather than the number of days since the repo was last used.
- [ ] The bot rounds the number of days the repo is inactive (ie 459.13 becomes 459).

### PR DESCRIPTION
**Pull request by junminahn**
closes #146

### CODE REVIEW

**Comment by junminahn** [comment]
@jleach @wenzowski added the fix for the issue https://github.com/bcgov/repomountie/issues/146

File: src/libs/repository.ts [hunk 1]
```diff
@@ -474,7 +474,7 @@ export const remindInactiveRepository = async (
       const text: string = await loadTemplate(TEXT_FILES.INACTIVE_REPO);
 
       await context.github.issues.create({
-        body: template(text)({ daysInactive: daysInactive.toFixed(2) }),
+        body: template(text)({ daysInactive: Math.round(daysInactive), inactiveDays: INACTIVE_DAYS }),
```
Comment (wenzowski) [inline src/libs/repository.ts:477]:
```suggestion
        body: template(text)({ daysInactive: Math.round(daysInactive), daysInactiveLimit: INACTIVE_DAYS }),
```

File: templates/inactive_repo_reminder.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 This issue is a kind reminder that your repository has been inactive for <%- daysInactive %> days. Some repositories are maintained in accordance with business requirements that infrequently change thus appearing inactive, and some repositories are inactive because they are unmaintained.
 
-To help differentiate products that are unmaintained from products that do not require frequent maintenance, repomountie will open an issue whenever a repository has not been updated in <%- daysInactive %> days.
+To help differentiate products that are unmaintained from products that do not require frequent maintenance, repomountie will open an issue whenever a repository has not been updated in <%- inactiveDays %> days.
```
Comment (wenzowski) [inline templates/inactive_repo_reminder.md:3]:
```suggestion
To help differentiate products that are unmaintained from products that do not require frequent maintenance, repomountie will open an issue whenever a repository has not been updated in <%- daysInactiveLimit %> days.
```

**Review by wenzowski — APPROVED** [review]
I'd tweak the variable naming a bit to make sure we avoid future confusion. Made a suggestion; not necessarily right. Don't want suggestion for improvement to block a PR that meets definition of done though so marking Approved so you have discretion on adjusting var naming.

File: src/libs/repository.ts (hunk 1 above)
Comment (junminahn) [reply src/libs/repository.ts]: @wenzowski great point! updated.

File: templates/inactive_repo_reminder.md (hunk 2 above)
Comment (junminahn) [reply templates/inactive_repo_reminder.md]: @wenzowski great point! updated.

**Review by jleach — APPROVED** [review]

**Pull request merged**